### DISCUSSION TITLE
Add breadcrumbs to the sidepanel

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13800] [Feature Requests]
When creating an entity through a relation, the side panel opens. If you do the same in the side panel, another side panel opens on top of it. This can get confusing when you have multiple 'level's that you've worked through, so how about using breadcrumbs in the side panel to show the user where in the 'tree' of relations you are? e.g.

`New Company >> New Employee >> New Address`

**Comment by azrikahar** [comment]
Just to add more info, there was a couple of discussions around this, which sparked the current stacked drawer approach added in #6440. This mockup relates more to the "breadcrumbs" in question: https://github.com/directus/directus/discussions/8635#discussioncomment-1448753

References: #8635 #4220

**Reply by benhaynes** [reply]
Thanks, Azri — I still think that mockup makes sense for the layered breadcrumb since that area is unused and there could be _many_ layers.

**Reply by sxsws** [reply]
Well...thinking ahead, if horizontal tabs are to be implemented for forms, this would lead to the side panel having to be of variable width to present enough horizontal room for a form with tabs when one is created through a relation, so I was thinking more underneath the title of the current side panel as that space would eventually be used.

**Reply by benhaynes** [reply]
I think the more important solution is making tabs work on viewports with limited horizontal width. With our Data Studio app being responsive, we'll already need to solve for that... so it shouldn't be an issue. Also, the drawer is set to the width of the main form size... so it's already standardized. That said, I'm not against making it flexible... we just did this for the nav sidebar, so we could implement in the drawer as well.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
